version/no_check.go

```diff
@@ -18,24 +18,10 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-// +build thriftrw.disableVersionCheck
-
 package version
 
-// CheckCompatWithGeneratedCodeAt will panic if the ThriftRW version used to
-// generated code (given by `genCodeVersion`) is not compatible with the
-// current version of ThriftRW.
-// This function is designed to be called during initialization of the
-// generated code.
-//
-// Rationale: It is possible that the old generated code is not compatible with
-// the new version of ThriftRW in subtle ways but still compiles successfully.
-// This function will ensure that the version mismatch is detected and help
-// avoid bugs that could be caused by this discrepancy.
-func CheckCompatWithGeneratedCodeAt(string, string) {
-	// This version does not do anything. It is used only when the
-	// disableVersionCheck is used. Use the following command to build a
-	// binary with this tag.
-	//
-	// 	go build -tags thriftrw.disableVersionCheck
```

Comment:
As a result of this, you can also delete the `-tags` business from,

https://github.com/thriftrw/thriftrw-go/blob/c6d9372002b8de6db0e9ee21549c2e2d83304889/Makefile#L65
https://github.com/thriftrw/thriftrw-go/blob/c6d9372002b8de6db0e9ee21549c2e2d83304889/gen/testdata/Makefile#L15
https://github.com/thriftrw/thriftrw-go/blob/c6d9372002b8de6db0e9ee21549c2e2d83304889/internal/plugin/gen.sh#L27